lib/python/treadmill_aws/hostmanager.py

```diff
@@ -48,6 +48,43 @@ def _instance_user_data(hostname, otp, instance_vars):
     )
 
 
+def _configure_dns(ipa_client, hostname, domain, ipaddr):
+    """Update IPA A and PTR records to match new host IP"""
+
+    if domain in hostname:
```

Comment:
nit - is it not

hostname = hostname.rstrip('.').rstrip(domain).rstrip('.') 

cause technically if domain is in the middle of the hostname, you will replace it with nothing... 

i am not sure what is more/less readable.